src/components/HelperErrorText/HelperErrorText.tsx

```diff
@@ -23,7 +23,7 @@ interface HelperErrorTextProps {
     /** Optional baseClass for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
     baseClass?: string;
     /** Added prop when HelperText is errored */
-    ariaLive?: boolean;
+    ariaLive?: "polite" | "off" | "assertive";
```

Comment:
Do we want this to be knob-able?

Reply:
Yeah, I think we should... I can push up that change

Reply:
I had trouble setting up a knob for this because of the type.